core/domain/config_domain.py

```diff
@@ -220,31 +230,31 @@ def __init__(self, name, schema, description, default_value):
         Registry.init_config_property(self.name, self)
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Returns the name of the configuration property."""
 
         return self._name
 
     @property
-    def schema(self):
+    def schema(self) -> Dict[str, Any]:
         """Returns the schema of the configuration property."""
 
         return self._schema
 
     @property
-    def description(self):
+    def description(self) -> str:
         """Returns the description of the configuration property."""
 
         return self._description
 
     @property
-    def default_value(self):
+    def default_value(self) -> Any:
```

Comment:
Explain why this returns `Any`

Reply:
As default_value of `configproperty` object can be any of these (int, bool, list. list[dict]), that's why there is any!

https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L374
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L382
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L401
https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/config_domain.py#L405

Reply:
Please add this explanation as a code comment.

Reply:
done!

Reply:
done!